WC.cs

```diff
@@ -0,0 +1,14 @@
+
+using System.Collections.Generic;
+
+namespace Wishlist{
+    public static class WC
+    {
+        public const string ProdcutTable = "products";
+        public const string UserTable = "users";
+        public const string  ItemTable = "wishitems";
+        public const string  RoleCustomer="customer";
+        public const string  RoleAdmin="admin";
+        public const string ProdutBlog="products";
+    }
+}
```

Comment:
I really like how it did this, it keeps things organised! 

We usually create ours and inject it with dependency injection. When you have some free time come to my desk and I will show you how.

Reply:
Actually you might we able to implement it using this doc : https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/options?view=aspnetcore-6.0

But let me know if that is confusing - it is slightly different than how we do it